src/nnue/nnue_accumulator.cpp

```diff
@@ -187,7 +187,7 @@ void AccumulatorStack::backward_update_incremental(
     const Square ksq = pos.square<KING>(Perspective);
 
     for (std::size_t next = m_current_idx - 2; next >= end; next--)
-        update_accumulator_incremental<Perspective, BACKWARD>(
+        update_accumulator_incremental<Perspective, false>(
```

Comment:
at least from a callers perspective the enum approach is easier to understand

Reply:
Yes slightly but is it worth defining new kinds of enums and having the logic in update_accumulator_incremental() be controlled by two separate variables that logically actually represent just one thing?  Let me know if you want me to revert the enum part of this PR.

Reply:
i have no strong feelings with that one, so i'll go with what's proposed